Tools/scripts/annotate_params_unittest.py

```diff
@@ -0,0 +1,536 @@
+#!/usr/bin/python3
```

Comment:
This one should go elsewhere - perhaps `Tools/autotest/unittest`?

Reply:
Done